paper/paper.md

```diff
@@ -0,0 +1,323 @@
+  - name: Shelly Johnson
+    orcid:
+    affiliation: 3
+  - name: Morgan Oneka
+    orcid:
+    affiliation: 1
+  - name: Sarah Tomkovich
+    orcid:
+    affiliation: 2
+  - name: Jacqueline Moltzau Anderson
+    orcid: 0000-0003-1398-5980
+    affiliation: 2
+  - name: Sarah K. Lucas
+    orcid:
+    affiliation: 2
+  - name: Patrick D. Schloss^[corresponding author]
+    orcid: 0000-0002-6935-4275
+    affiliation: 2
+affiliations:
+  - name: Department of Computational Medicine & Bioinformatics, University of Michigan
+    index: 1
+  - name: Department of Microbiology & Immunology, University of Michigan
+    index: 2
+  - name: TODO
+    index: 3
+  - name: Independent educator
+    index: 4
+date: 27 Apr. 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Inspired by well-established material and pedagogy provided by The Carpentries
+[@wilson_software_2016], we developed a two-day workshop curriculum that teaches
+introductory R for plotting and data analysis, the Unix shell, version control
+with git, and GitHub.
+While the official Software Carpentry curriculum is extensive, we found
+that it contains too much content for the time allotted in a two-day workshop,
+and the independent nature of the lessons leaves learners confused about how to
+integrate the skills together in their own work.
+The new curriculum we developed (https://umcarpentries.org/intro-curriculum-r/), which is hosted on GitHub,
+aims to teach novices how to implement reproducible research practices to
+perform data analysis.
+It emphasizes the iterative nature of coding and data analysis through
+integrated live coding lessons and practice exercises.
+Furthermore, this curriculum provides a succinct resource for learners to reference both during and after
+the workshop, and lowers the barrier to entry for new instructors as they do not
+have to develop their own curriculum or sift through extensive content.
+We developed this curriculum during a two-day sprint,
+successfully used the curriculum to host a two-day virtual workshop,
+and updated the material based on instructor and learner feedback.
+We hope that this curriculum will be useful to other instructors interested in
+teaching workshops with similar learning objectives.
+
+# Statement of Need
+
+The University of Michigan instance of The Carpentries has taught workshops
+using the curriculum created by The Carpentries organization since 2015.
+In that time, our instructors have found several advantages and disadvantages to
+using the Software Carpentry curriculum in particular.
+Some advantages are that any programming language lesson (e.g. R or
+Python) can be paired with lessons on the Unix shell and version control,
+the lessons have been refined over years by many contributors and taught at
+workshops around the world, and
+the instructional design demonstrates good pedagogy for teaching beginner data
+science practitioners.
+
+However, we found that the curriculum did not meet all of our needs.
```

Comment:
It seems odd to me to jump right to "the [existing Carpentries] curriculum did not meet all of our needs" without first listing those exact needs. This seems to defining the needs negatively rather than positively asserting them. Is the next paragraph meant to be the positive statement of what exactly the needs were? I don't think the existing carpentries curriculum has no use, it's just perhaps that we had different goals than they may have had when preparing the materials and just making that point clear.